workflow/setup_expt.py

```diff
@@ -224,7 +224,15 @@ def fill_COMROT_forecasts(host, inputs):
     """
     Implementation of 'fill_COMROT' for forecast-only mode
     """
-    print('forecast-only mode treats ICs differently and cannot be staged here')
+    if inputs.system in ['gfs']:
+        print('forecast-only mode treats ICs differently and cannot be staged here')
+    elif inputs.system in ['gefs']:  # Temporarily copy ICs from icsdir into COM for      testing
+        print('temporary hack to stage gefs ICs for testing')
+        comrot = os.path.join(inputs.comrot, inputs.pslot)
+        idatestr = datetime_to_YMDH(inputs.idate)
+        current_cycle_dir = f'gefs.{idatestr[:8]}}'
```

Comment:
```suggestion
        current_cycle_dir = f'gefs.{{idatestr[:8]}}'
```
Missed right handed escaped bracket for single format string

Reply:
addressed